src/object-properties.js

```diff
@@ -55,6 +55,15 @@ const properties = {
    * @type {(string|StringExpression)=}
    */
   footnote: '',
+  /**
+   * totals settings
+   * @type {object}
+   */
+  totals: {
+    show: true,
+    position: 'noTotals',
+    label: 'Totals',
+  },
```

Comment:
@niekvanstaveren How do we want to do this? If we add theme here for now, do we need to add it to the spec as well, at least before releasing?

Reply:
Yes I think so